api/base/filters.py

```diff
@@ -541,10 +541,12 @@ def param_queryset(self, query_params, default_queryset):
         if filters:
             for key, field_names in filters.iteritems():
                 for field_name, data in field_names.iteritems():
-                    if isinstance(queryset, list):
-                        queryset = self.get_filtered_queryset(field_name, data, queryset)
-                    else:
-                        queryset = self.filter_by_field(queryset, field_name=field_name, operation=data)
+                    operations = [item for item in data] if isinstance(data, list) else [data]
```

Comment:
This seems weird to me. If data is a list, you list iterate over the list items to cast it to a list? If it were a generator, that makes sense. Wouldn't it be `operations = data if isinstance(data, list) else [data]`?